wasserstein/_wasserstein.py

```diff
@@ -28,9 +28,9 @@
 import wasserstein.config
 
 if wasserstein.config.openmp():
-	from ._wasserstein_omp import *
+	from .omp._wasserstein import *
 else:
-	from ._wasserstein_noomp import *
+	from .noomp._wasserstein import *
```

Comment:
I forget. Is there actually a situation in which `noomp` ever gets built? Or have we just ensured that there is an `omp` now? If `wasserstein.noomp` can never exist then we should remove it.

If it does exist, can you direct me to where in the code it gets created?

Reply:
Ah, as the `CMakeLists.txt` has

```cmake
find_package(
	OpenMP
	REQUIRED)
```

then I don't think there is a way to build without OpenMP now, and so 

```python
if wasserstein.config.openmp():
	from .omp._wasserstein import *
else:
	from .noomp._wasserstein import *
```

can just get reduced to

```python
if wasserstein.config.openmp():
	from wasserstein.omp._wasserstein import *
```